cookieless_sessions.gemspec

```diff
@@ -23,15 +23,15 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "redis-session-store", "~> 0.7"
 
-  spec.add_development_dependency "bundler", "~> 1.5"
+  spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake", "~> 10.0"
```

Comment:
Hier gerne bei der Gelegenheit die Einschränkung auf `"~> 10.0"` aufheben. Die führt nämlich gerade zu 'nem  Dependabot-Alert und dem Hinweis, dass man aus Sicherheitsgründen auf rake >= 12.3.3 updaten sollte.

Reply:
Allgemein würde ich davon ausgehen, dass es besser ist, die meisten Versionseinschränkungen in den development dependencies aufzuheben. Die sehen nämlich alle schon recht veraltet aus und ich gehe davon aus, dass sie nicht nötig sind. Also bei der Gelegenheit hier gerne wie bei frikandel einmal alle bis auf rspec-rails >= 3.0 entfernen. (Letzteres ist nötig, weil es bei der Spec Syntax breaking changes zwischen rspec 2 und rspec 3 gibt.)